exp-player/addon/styles/components/exp-overview.scss

```diff
@@ -1,5 +1,6 @@
 .format-average-label {
-    margin-left: 10px;
-    position: absolute;
+    display: block;
+    width: 100%;
+    margin-left: calc(50% + 10px);
```

Comment:
display block / width 100% makes it assume the width of the parent, then we need to move it over 50% + the extra margin of the radio button.

See updated main comment for other languages